c7n/filters/iamaccess.py

```diff
@@ -211,9 +211,17 @@ def normalize_conditions(self, s):
     # Condition handlers
 
     # kms specific
+    # https://docs.aws.amazon.com/kms/latest/developerguide/policy-conditions.html#conditions-kms
     def handle_kms_calleraccount(self, s, c):
+        raise Exception(self.allowed_accounts)
         return bool(set(map(_account, c['values'])).difference(self.allowed_accounts))
 
+    def handle_kms_viaservice(self, s, c):
+        return False
```

Comment:
can you explain why this is returning False unconditionally? this doesn't look right

Reply:
The other cross account handlers return False when they are compliant, rather than the more intuitive value of True. 

Without this fix, the presence of these condition keys causes custodian to pick up statements containing these condition keys; print a warning that it cant handle them correctly, and treat them as things to mark / remediate.

I think Custodian shouldn't care about the ViaService or GrantOperations tags being present, they don't relate to any of the whitelist options for cross-account.